src/Kafka/Settings/Settings.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Modal,
+  ModalVariant,
+  Page,
+  PageSection,
+  Spinner,
+  Switch,
+} from "@patternfly/react-core";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { SettingsAlert } from "./components/SettingsAlert";
+import type { AlertStatus, SettingsStatus } from "./types";
+
+export type SettingsProps = {
+  connectionStatus: SettingsStatus;
+  onSwitchClick: () => void;
+  isModalOpen: boolean;
+  onClickTurnOff: () => void;
+  alertStatus: AlertStatus;
+  connectionState: boolean;
+  onClickClose: () => void;
+};
+
+export const Settings: FunctionComponent<SettingsProps> = ({
+  connectionStatus,
+  onSwitchClick,
+  isModalOpen,
+  onClickTurnOff,
+  alertStatus,
+  onClickClose,
+  connectionState,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  return (
+    <>
+      <PageSection>
+        <Card>
+          <CardBody>{t("settings.kafka_instance_settings_label")}</CardBody>
+        </Card>
+      </PageSection>
+      <Page>
+        <PageSection>
+          <Card>
+            <CardBody>
+              <Flex>
+                <FlexItem component={"span"} spacer={{ default: "spacerSm" }}>
```

Comment:
I previously shared this feedback from PF dev on how to handle the responsive layout. 

> I would just use a flex layout set to nowrap with the text flex child set to flex grow - or you could use a split layout with the text flex child set to "fill" (.pf-m-fill or the isFilled react prop)

These were the suggestions they made which I don't see applied here:
- use the nowrap prop on the flex layout
- use the flex grow prop on the flex item that contains the text
  - Also note, all text should be in a single flex item so that the text contents wrap as shown  in [the responsive example](https://docs.google.com/document/d/1hO_aMtRV75F48d4PV1j_LY1GMVXgFKwANeUwvJTzMoQ/edit?disco=AAAAavQirGA). Instead, you have separate text strings in separate flex items. 

The exact props to use should be noted in the PF react component documentation for the flex layout. If you're struggling to figure out what props to use where, please reach out.

Reply:
One follow-up comment on this, you can have a flex layout nested inside a flex layout. For example:
```
<Flex> 👈 use the nowrap prop here
  <FlexItem> 👈 use the flex grow prop here
    <Flex>
      <FlexItem component={"span"} spacer={{ default: "spacerSm" }}>...</FlexItem> 
       {include all text flex items here}
    </Flex>
  </FlexItem>
  <FlexItem> {include switch here} </FlexItem>
</Flex>
```